### DISCUSSION TITLE
How to: Split-Release album tagging

### DISCUSSION THREAD

**Discussion by hubortje** [navidrome/navidrome#3250] [Q&A] [answered]
Hey there,

I'm facing an "issue" in my collection, which I've solved in a way I'm not really happy with, and I'd be glad to hear your solutions, if you've solved this way smarter.

# Background
My collection is saved in a folder structure like e.g. `Music\Band A\Album 1\` and `Music\Band B\Album 1`.
Some of those bands have so-called **Split**, releases, where both bands are featured with their own respective songs, e.g. `Band A/Band B Split`.

I'd like to store their respective songs in their own folders, like `Band A\Band A - Band B\01 Song of Band A\` and `Band B\Band A - Band B\02 Song of Band B\`.
# My issue
I've tagged songs the following way:
```
Title: Song of Band A
Interpret: Band A
Album: Band A - Band B
Date: 2024
Number: 1
Genre: Awesome Music
Album interpret: Band A
Cover: cover.jpg
```
and
```
Title: Song of Band B
Interpret: Band B
Album: Band A - Band B
Date: 2024
Number: 1
Genre: Awesome Music
Album interpret: Band B
Cover: cover.jpg
```
Now, Navidrome does not recognize both songs are used in one single release. Obviously, the `Album interpret`-tag makes them both their own releases. So, I've deleted the `Album interpret`-tag, and added the `Compilation`-tag set to 1.
Now I've got both songs in one album, with their respective interpret, but the album is from `Various artists` and does not appear in the list of `Band A` nor `Band B`.
# Goal
I'd like to have both songs merged into one album, which then is found in the list of both artists.
# Questions
1. Did you face the same issue and were you able to solve it?
2. Is it possible to add a list of separate tags into one tag? If so, what is the delimiter?

Thank you very much!

**Comment by dweymouth** [comment, marked as answer]
You will have to wait until https://github.com/navidrome/navidrome/pull/2709 is completed and released to have a single release show up in the album list for both artists. But you can tag it in a way that will be compatible with that refactoring, by having all the tracks of the album share A) the same ALBUM tag; B) the same ALBUMARTIST tag(s) - you can either tag a single ALBUMARTIST with the ';' separator between the two names, or else each track can have two ALBUMARTIST tags -one with each artist name; and B) the same release date /original release date tags if present. For now, if you go with the 2 ALBUMARTIST tags, Navidrome will currently only process the first one, and if you go with the single tag with ; separator, it will likely process it as a single artist with that name (eg "Artist A; Artist B")

**Reply by hubortje** [reply]
Thank you! Then I'll wait patiently. :-)
